lib/Controller/ApiController.php

```diff
@@ -73,12 +73,13 @@ public function __construct(IRequest $request,
 	 *
 	 * Set the download limit for a given link share
 	 */
-	public function setDownloadLimit(string $token, int $limit): Response {
+	public function setDownloadLimit(string $token, $limit): Response {
```

Comment:
I'd stick to the int typing on the limit parameter.

Reply:
If `null` needs to be an option you can mark it as nullable

```suggestion
	public function setDownloadLimit(string $token, ?int $limit = null): Response {
```

Reply:
You actually do not need the = null, just the ?int.